pkgs/top-level/python-packages.nix

```diff
@@ -1652,7 +1652,9 @@ in {
 
   cvxopt = callPackage ../development/python-modules/cvxopt { };
 
-  cvxpy = callPackage ../development/python-modules/cvxpy { };
+  cvxpy = callPackage ../development/python-modules/cvxpy {
+    inherit (pkgs.llvmPackages) openmp;
```

Comment:
Does this mean that we're using the clang OpenMP even if the package is built with gcc?

Reply:
Ah, didn't know there was one built-in to GCC. Thanks, I'll fix that.